src/UbuntuToolkit/colorutils.cpp

```diff
@@ -28,4 +28,40 @@ qreal ColorUtils::luminance(const QColor &color)
     return color.lightnessF();
 }
 
+/*
+ * Contrast calculation
+ * https://www.w3.org/TR/WCAG20-TECHS/G17.html#G17-procedure
+ *
+ * Based on the code by kirilloid
+ * https://stackoverflow.com/questions/9733288/how-to-programmatically-calculate-the-contrast-ratio-between-two-colors
+ */
+
+qreal ColorUtils::contrast(const QColor &color)
```

Comment:
Either rename this to something like "whiteContrast", or add a second parameter for the other colour. I would prefer the latter option, so that this function could be useful in more cases (dark themes, for example).

Reply:
Done